v0/destinations/indicative/transform.js

```diff
@@ -7,9 +7,71 @@ const {
   defaultPostRequestConfig
 } = require("../../util");
 
+const handleProperties = properties => {
+  const result = {};
+  let l;
+
+  const recurse = (cur, prop) => {
+    let i;
+    if (Object(cur) !== cur) {
+      // primitive types
+      result[prop] = cur;
+    } else if (Array.isArray(cur)) {
+      if (cur.length > 0) {
+        const fel = cur[0];
+        if (Object(fel) !== fel) {
+          result[prop] = cur.toString();
```

Comment:
I think this will fail for arrays with combination of primitives and objects. Should we handle that?

Reply:
Updated the array handling logic.